ads_savon/Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
```

Comment:
I don't think we're interested in using bundler at this time. Is there a particular advantage you see during development?